docs/guides/int_framework/samples/intro/groupmodule.cs

```diff
@@ -0,0 +1,15 @@
+[Group("group-name", "Group description")]
+public class CommandGroupModule : InteractionModuleBase<SocketInteractionContext>
+{
+    [SlashCommand("ping", "Get a pong")]
+    public async Task PongSubcommand()
+        => await RespondAsync("Pong!");
+
+    [Group("subcommand-group", "Subcommand group description")]
+    public class SubcommandGroupModule : InteractionModuleBase<SocketInteractionContext>
+    {
+        [SlashCommand("echo", "Echo an input")]
+        public async Task PongSubcommand(string input)
```

Comment:
I suggest naming this 'EchoSubcommand' instead for logic

Reply:
Oops, didn't notice that i forgot to change the name ._.